src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+@media screen and (max-width: 1024px) {
+  .container {
+    max-width: 1024px;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.header-logo__link {
+  text-decoration: none;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__link {
+  list-style: none;
+  margin-left: 20px;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: var(--primary-color);
+}
+
+[data-qa="hover"] {
```

Comment:
pls use only classNames to assign styles

Reply:
add `data-qa="hover"` attribute to the 4th link for testing (`Laptops & computers`)